javascript_nodejs/14.nlp-with-dispatch/index.js

```diff
@@ -6,7 +6,7 @@ const restify = require('restify');
 
 const CONFIG_ERROR = 1;
 
-// Import reuqired bot services. See https://ama.ms/bot-services to learn more about the different part of a bot
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different part of a bot
```

Comment:
grammar. "parts of a bot" (make part plural). Also add a "." to the end.

Reply:
Fixed.